### PR TITLE
API-4295 Make look for config file in current directory at first. Look in home directory if file wasn't found in current dir

### PR DESCRIPTION
**Pull request by inesusvet**
See JIRA https://datarobot.atlassian.net/browse/API-4295

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -158,7 +158,7 @@ Example::
 
 Using the configuration file
 ----------------------------
-The `batch_scoring` command checks for the existence of a batch_scoring.ini file at the location `$HOME/batch_scoring.ini` (your home directory) and the directory where you are running the script (working directory). If this file exists, the command uses the same arguments as those described above. If the file does not exist, the command proceeds normally with the command line arguments. The command line arguments have higher priority than the file arguments (that is, you can override file arguments using the command line).
+The `batch_scoring` command checks for the existence of a batch_scoring.ini file at the location `$HOME/batch_scoring.ini` (your home directory) and the directory where you are running the script (working directory). The working directory has higher priority over home directory. If this file exists, the command uses the same arguments as those described above. If the file does not exist, the command proceeds normally with the command line arguments. The command line arguments have higher priority than the file arguments (that is, you can override file arguments using the command line).
```
Comment (inesusvet) [inline README.rst]: Added
> The working directory has higher priority over home directory.

File: tests/test_conf_file.py [hunk 2]
```diff
@@ -1,42 +1,62 @@
...
-                                                 CONFIG_FILENAME)
-    finally:
-        os.remove(os.path.join(os.getcwd(),
-                               CONFIG_FILENAME))
+
+def test_file_from_working_directory__has_priority(
+        config_file_in_home_dir,
+        config_file_in_current_dir,
+):
+    assert get_config_file() == config_file_in_current_dir
```
Comment (inesusvet) [inline tests/test_conf_file.py]: The previous generalization shrinks test sizes to one line. I think it's good

File: README.rst (hunk 1 above)
Comment (madmott) [reply README.rst]: I think a phrasing of "checks for the existence of a batch_scoring.ini file at the directory where you are running the script (working directory) and, if it is not found in the working directory, in `$HOME/batch_scoring.ini` (your home directory)" might be clearer and more concise?

(You could then drop "The working directory has a higher priority over home directory".)

File: datarobot_batch_scoring/utils.py [hunk 3]
```diff
@@ -252,10 +252,10 @@ def get_config_file():
     """
     home_path = path_join(expanduser('~'), CONFIG_FILENAME)
     cwd_path = path_join(getcwd(), CONFIG_FILENAME)
-    if isfile(home_path):
-        return home_path
-    elif isfile(cwd_path):
+    if isfile(cwd_path):
         return cwd_path
+    elif isfile(home_path):
+        return home_path
```
Comment (inesusvet) [inline datarobot_batch_scoring/utils.py]: I am very interested how the coverage is calculated for this repo
> Coverage decreased (-0.05%) to 82.863%

I assert that
- there were no changes in lines-of-code of production code;
- there were no changes in the tests of logical paths through this code

**Comment by inesusvet** [comment]
Review comments have been addressed. I am asking for an approve

File: CHANGES.rst [hunk 4]
```diff
@@ -1,3 +1,10 @@
+1.13.3 (unreleased)
+====================
+
+Bugfixes
+--------
+* Change the sequence of config file look up: current directory first, then user's home directory
```
Comment (madmott) [inline CHANGES.rst]: I think “Changes” not Change.  Also maybe “A config file in the working directory will now take precedence over a config file in the user’s home directory.” Vs the current phrasing

**Pull request merged**